src/test/java/org/fcrepo/client/integration/FcrepoClientIT.java

```diff
@@ -342,8 +377,6 @@ public void testGetRange() throws Exception {
         final String mimetype = "text/plain";
         final String bodyContent = "Hello world";
         final FcrepoResponse response = client.post(new URI(serverAddress))
-                .digestMd5("f0ef7081e1539ac00ef5b761b4fb01b3")
```

Comment:
Leaving them out for the reasons you detailed seems right. Thanks for the explanation.